tests/forms_tests/widget_tests/test_multiplehiddeninput.py

```diff
@@ -68,8 +69,29 @@ def test_render_increment_id(self):
         self.check_html(
             self.widget, 'letters', ['a', 'b', 'c'], attrs={'id': 'hideme'},
             html=(
-                '<input type="hidden" name="letters" value="a" id="hideme_0">\n'
-                '<input type="hidden" name="letters" value="b" id="hideme_1">\n'
-                '<input type="hidden" name="letters" value="c" id="hideme_2">'
+                '<input type="hidden" name="letters_0" value="a" id="hideme_0">\n'
+                '<input type="hidden" name="letters_1" value="b" id="hideme_1">\n'
+                '<input type="hidden" name="letters_2" value="c" id="hideme_2">'
             ),
         )
+
+    def test_unique_names_single(self):
+        self.check_html(
+            self.no_unique_names, 'email', ['test@example.com'],
+            html='<input type="hidden" name="email" value="test@example.com">',
+        )
+
+    def test_unique_names_multiple(self):
+        self.check_html(
+            self.no_unique_names, 'email', ['test@example.com', 'foo@example.com'],
+            html=(
+                '<input type="hidden" name="email" value="test@example.com">\n'
+                '<input type="hidden" name="email" value="foo@example.com">'
+            ),
+        )
+
+    def test_unique_names_render_attrs(self):
```

Comment:
Do we really need this test? seems it doesn't relate to this change.

Reply:
Hi @hramezani 
I'm trying to show passing tests for both `unique_names = True` and `unique_name=False`. Would it be preferable to reduce it to just a single test for `unique_names=False`. Appreciate it's a bit repetitive.

Reply:
Hi @smithdc1, I meant ‍`test_unique_names_render_attrs‍`. Did your changes affect on attrs rendering?

Reply:
Hi @hramezani no change to attrs rendering. I added a test to prove that this is the case. Do you suggest that this test is not required?

Reply:
I think we don't need it. but lets @felixxm decide about it.
Thanks for the patch :+1: